### PR TITLE
Migrate organization details to SDK 

### PR DESCRIPTION
**Pull request by selankon**
- [x] Rebase https://github.com/vocdoni/explorer-ui/pull/131/

### CODE REVIEW

File: components/pages/organizations/details/index.tsx [hunk 1]
```diff
@@ -6,37 +6,36 @@ import { Grid, Column } from '@components/elements/grid';
...
 
   return (
     <PageCard>
-      <CardImageHeader
-        title={metadata?.name.default}
-        processImage={metadata?.media.header}
-        // subtitle={entity?.name.default}
-        entityImage={metadata?.media.avatar}
-      />
+      <CardImageHeader title={orgName} processImage={header} entityImage={avatar} />
```
Comment (elboletaire) [inline components/pages/organizations/details/index.tsx:30]: I think you should consider replacing or updating this component with the `OrganizationAvatar` component.

File: components/pages/organizations/details/index.tsx [hunk 2]
```diff
@@ -6,37 +6,36 @@ import { Grid, Column } from '@components/elements/grid';
...
 
-      {metadata?.description.default && (
+      {description && (
         <Grid>
           <Column sm={12}>
             <Typography variant={TypographyVariant.Body1}>
               {i18n.t('organizations.details.organization_description')}
             </Typography>
-            <Typography variant={TypographyVariant.Small}>{metadata?.description.default}</Typography>
+            <Typography variant={TypographyVariant.Small}>{description}</Typography>
```
Comment (elboletaire) [inline components/pages/organizations/details/index.tsx:38]: You could be using `OrganizationDescription` here

**Review by elboletaire — CHANGES_REQUESTED** [review]
You should consider using the provided Organization components in our chakra-components package. you would probably end up removing some components that you might not need anymore. Actually, the Image component used in explorer-ui is an exact copy of the one I created for vocdoni-app back in the day, and the one provided in chakra-components is a similar version (it uses an IPFS proxy URL instead of direct api calls to IPFS, which was cool in part, but at the same time was also overcomplicated).

So, in the case of the `OrganizationAvatar`, you can either chose to create a component based on that one, or you could try to replace the Image component you have, by the one provided by the chakra-components package.

File: components/pages/organizations/details/index.tsx (hunk 2 above)
Comment (selankon) [reply components/pages/organizations/details/index.tsx]: https://github.com/vocdoni/explorer-ui/pull/133/files#diff-351efab9aa5fb2447dcbcbd9f0a72dd7a5a1082cb614f5689174e0f16ad24131R39

**Pull request merged**